sealer/cmd/delete.go

```diff
@@ -27,8 +29,29 @@ var deleteCmd = &cobra.Command{
 	Use:     "delete",
 	Short:   "delete a cluster",
 	Long:    `if provider is BARESERVER will delete kubernetes nodes, or if provider is ALI_CLOUD, will delete all the infra resources`,
-	Example: `sealer delete -f /root/.sealer/mycluster/Clusterfile`,
+	Example: `sealer delete -f /root/.sealer/mycluster/Clusterfile [--force]`,
 	Run: func(cmd *cobra.Command, args []string) {
+		force, err := cmd.Flags().GetBool("force")
+		if err != nil {
+			logger.Error(err)
+			os.Exit(1)
+		}
+		if !force {
+			var yesRx = regexp.MustCompile("^(?:y(?:es)?)$")
+			var noRx = regexp.MustCompile("^(?:n(?:o)?)$")
+			var input string
+			for {
+				fmt.Printf("Are you sure you want to delete the cluster? Yes [y/yes], No [n/no] : ")
```

Comment:
Are you sure you want to delete the cluster? -> Are you sure to delete the cluster?

Reply:
i get it

Reply:
LGTM. if rebase the commit.